src/lxc/cgroups/cgroup.h

```diff
@@ -28,6 +28,7 @@
 #include <stddef.h>
 #include <sys/types.h>
 
+#define DEFAULT_MOUNTPOINT "/sys/fs/cgroup"
```

Comment:
Shouldn't that have CGROUP in the name to avoid potential later clashes with other things that may have a default mount point?

Reply:
myeah